### PR TITLE
👍 update for new shopify-express api

### PR DESCRIPTION
**Pull request by TheMallen**
Please try this out on your own dev stores :)

### CODE REVIEW

**Comment by AWaselnuk** [comment]
I can't 🎩because my `redirect_uri` is not whitelisted during the oauth dance. In other words the way I was setting up a proxy to `localhost` seems to suddenly not work.

I'll try again tomorrow :(

**Comment by TheMallen** [comment]
Ok, thanks, I'll take a look at this again tonight or tommorrow morning

**Comment by TheMallen** [comment]
@AWaselnuk @lopert  is right that we need to also change the README, so that's probably why your whitelist is wrong.

@lopert You're right that the readme needs updating, but I'm not seeing issues with the escaping locally. We should chat about how this is set up for you. For me it works fine with latest `@shopify/shopify-express`

**Comment by jamiemtdwyer** [comment]
> &redirect_uri=https%3A%2F%2Falexbradley.ngrok.io%2F%2Fshopify%2Fauth%2Fcallback

If you mean this is what appears in your address bar during the OAuth grant screen, this is actually intended. The URI should be escaped since it's being passed as a URL param. Shopify knows that it should unescape it on its side.

**Comment by AWaselnuk** [comment]
Sidenote: our UI demo is pretty lame - we don't even display a flash or UI message of any kind when you get a `404`. It would be so great if someone could tackle https://github.com/Shopify/shopify-node-app/issues/75

**Comment by TheMallen** [comment]
@AWaselnuk updated the URL

File: server/shopify.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  shopStore,
+  host: SHOPIFY_APP_HOST,
+  apiKey: SHOPIFY_APP_KEY,
+  secret: SHOPIFY_APP_SECRET,
+  callbackRoute: CALLBACK,
+  scope: ['write_orders, write_products'],
+  afterAuth(request, response) {
+    const { session: { accessToken, shop } } = request;
+
+    // registerWebhook(shop, accessToken, {
```
Comment (AWaselnuk) [inline server/shopify.js]: Can we remove these comments + `console.log`?

**Pull request merged**